practice-one/src/components/introduction.html

```diff
@@ -1,22 +1,35 @@
 <section class="introduction-container">
   <div class="introduction-content-container">
-     <div class="people-trust-icon">
-        <img src="./assets/images/trust.svg" alt="Heart icon" />
-        People Trust us
-     </div>
-     <h1 class="header-introduction">
-         We're 
-         <span class="primary-color"> Serious </span>
-         For 
-         <br/>
-         <span class="primary-color">Food</span> 
-         & 
-         <span class="accent-color">Delivery</span>.
-     </h1>
-     <img class="header-decor" src="./assets/images/decor.png" alt="decor flash">
-     <p class="para-introduction">
+    <div class="people-trust-icon">
+      <img src="./assets/images/trust.svg" alt="Heart icon" />
+      People Trust us
+    </div>
+    <h1 class="header-introduction">
+      We're 
+      <span class="primary-color"> Serious </span>
+      For 
+      <br/>
+      <span class="primary-color">Food</span> 
+      & 
+      <span class="accent-color">Delivery</span>.
+    </h1>
+    <img class="header-decor" src="./assets/images/decor.png" alt="decor flash">
+    <p class="para-introduction">
       Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.
     </p>
+    <div class="search-food-container">
+      <img src="./assets/images/search-icon-black.svg">
+      <input type="text" placeholder="Search food">
+      <img src="./assets/images/search-icon-white.svg">
```

Comment:
for empty tag should have `/` at the end

Reply:
i understood , thanks